<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
    color: #333333;
    background-color: #f5f5f5;
}</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>781</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: #ffffff;
}

QTableWidget::item {
    padding: 8px;
}

QTableWidget::item:selected {
    background-color: #d9e4eb;
    color: #333333;
}

QHeaderView::section {
    background-color: #9eb3c2;
    color: #ffffff;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>611</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #d9d9d9;
    border-radius: 4px;
    padding: 6px 12px;
    background-color: #ffffff;
}

QLineEdit:focus {
    border-color: #9eb3c2;
    outline: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-weight: bold;
    margin-bottom: 8px;
}</string>
       </property>
       <property name="text">
        <string>Task Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-weight: bold;
    margin-bottom: 8px;
}</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #d9d9d9;
    border-radius: 4px;
    padding: 6px 12px;
    background-color: #ffffff;
}

QLineEdit:focus {
    border-color: #9eb3c2;
    outline: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilos para QGroupBox */
QGroupBox {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    padding: 16px;
    margin-top: 16px;
    font-weight: bold;
    background-color: #e6eef4;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
    color: #333333;
    background-color: #d9e4eb;
    border: 1px solid #9eb3c2;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}</string>
    </property>
    <property name="title">
     <string>Processing</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Open File 1 - Daily</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string> Open File 2 - Database</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Run Check</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Save Run Changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilos para QGroupBox */
QGroupBox {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    padding: 16px;
    margin-top: 16px;
    font-weight: bold;
    background-color: #e6eef4;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
    color: #333333;
    background-color: #d9e4eb;
    border: 1px solid #9eb3c2;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>277</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Show Assets</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Generate Reports</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Show Database</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #9eb3c2;
    color: #333333; /* Letras de los botones en gris oscuro */
    padding: 8px 16px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8aa1b4;
}

QPushButton:pressed {
    background-color: #7b92a7;
}
</string>
        </property>
        <property name="text">
         <string>Show Daily</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilos para QGroupBox */
QGroupBox {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    padding: 16px;
    margin-top: 16px;
    font-weight: bold;
    background-color: #e6eef4;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 8px;
    color: #333333;
    background-color: #d9e4eb;
    border: 1px solid #9eb3c2;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}</string>
    </property>
    <property name="title">
     <string>Insights</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>105</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdNumber_3">
        <property name="styleSheet">
         <string notr="true">/* Estilos para QLCDNumber */
QLCDNumber {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    color: #333333;
    background-color: #e6eef4;
    padding: 4px;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="styleSheet">
         <string notr="true">/* Estilos para QLCDNumber */
QLCDNumber {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    color: #333333;
    background-color: #e6eef4;
    padding: 4px;
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vulnerabilities</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Solved</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="styleSheet">
         <string notr="true">/* Estilos para QLCDNumber */
QLCDNumber {
    border: 1px solid #9eb3c2;
    border-radius: 4px;
    color: #333333;
    background-color: #e6eef4;
    padding: 4px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
